### PR TITLE
Add SonarQube badges and Update tools documentation

### PR DESCRIPTION
**Pull request by jujaga**
# Description

This PR adds SonarQube badges and updates the tools documentation regarding Jenkins and SonarQube setup.

[SHOWCASE-81](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-81)
[SHOWCASE-191](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-191)

## Types of changes

Documentation (non-breaking change with enhancements to documentation)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-67

File: tools/README.md [hunk 1]
```diff
@@ -12,13 +16,13 @@ See tools/docker/contrib/jenkins/configuration/init.groovy.d/003-create-jobs.gro
 
 To add or change the jobs, this is where you want to go.  The name of this file is important, as it needs to get run *BEFORE* the 003-register-github-webhooks.groovy included in the basic install.  Scripts are run alphabetically.  The jobs need to be created before the github webhooks are created.  Our jobs script will read secrets and configmaps created during this setup; described below.
 
-## Prerequisites
+### Prerequisites
 
 You should have your 4 OpenShift projects created for you by ???: dev, test, prod, tools.
```
Comment (parc-jason) [inline tools/README.md]: really need to find out ??? or remove this text.

File: tools/README.md [hunk 2]
```diff
@@ -27,131 +31,194 @@ Change as necessary...
 Fill in this documentation about parameters and labels from pipeline-cli...
```
Comment (parc-jason) [inline tools/README.md]: we should remove all of this if we are not using it. it's more confusing than helpful if we don't use those variables/parameters.

File: tools/README.md (hunk 1 above)
Comment (jujaga) [reply tools/README.md]: Changed wording to assume the structure instead.

File: tools/README.md (hunk 2 above)
Comment (jujaga) [reply tools/README.md]: Done

**Pull request merged**